src/cli/README_COAP.md

```diff
@@ -105,6 +121,20 @@ Done
 Done
 ```
 
+### obs \<address\> \<uri-path\> \[type\]
```

Comment:
shall we use `observe` for readability?

Reply:
We certainly can do that… I was thinking along the lines of keeping strings short but 4 bytes is hardly a big expense.